src/lambda/update-syllabus/utils.py

```diff
@@ -0,0 +1,72 @@
+from const import *
+
+class course():
+    def __init__(self,data):
+        self.id = data["a"]
+        self.data = data
+
+def create_db_item(new_course,school):
+    Item = {
+        s3_to_dynamo["a"] : new_course.data["a"],
+        s3_to_dynamo["b"] : new_course.data["b"],
+        s3_to_dynamo["c"] : new_course.data["c"],
+        s3_to_dynamo["d"] : new_course.data["d"],
+        s3_to_dynamo["e"] : new_course.data["e"],
+        s3_to_dynamo["f"] : new_course.data["f"],
+        s3_to_dynamo["g"] : new_course.data["g"],
+        s3_to_dynamo["h"] : new_course.data["h"],
+        s3_to_dynamo["i"] : new_course.data["i"],
+        s3_to_dynamo["j"] : new_course.data["j"],
+        s3_to_dynamo["k"] : new_course.data["k"],
+        s3_to_dynamo["l"] : new_course.data["l"],
+        s3_to_dynamo["m"] : new_course.data["m"],
+        s3_to_dynamo["n"] : new_course.data["n"],
+        s3_to_dynamo["o"] : new_course.data["o"],
+        s3_to_dynamo["p"] : new_course.data["p"],
+        "school" : school
+    }
+    return Item
+
+def update_course(new_course,table,school):
+    #test
+    print("update course{}",new_course.data['a'])
+    response = table.update_item(
+            Key={
+                'id' : new_course.data["a"]
+            },
+            UpdateExpression="set title=:b, title_jp=:c, instructor=:d, instructor_jp=:e, \
```

Comment:
Use capital letters for the command
```suggestion
            UpdateExpression="SET title=:b, title_jp=:c, instructor=:d, instructor_jp=:e, \
```

Reply:
Fixed.